include/ideep/operators/inner_product.hpp

```diff
@@ -3,43 +3,107 @@
 
 namespace ideep {
 
+// Note:
+// Inner product does not support quantization.
+// Please switch to matmul for quantized *mm ops
+
+struct inner_product_forward_params {
+  dnnl::inner_product_forward::primitive_desc _pd;
+  dnnl::inner_product_forward _primitive;
+  attr_t _op_attr;
+  attr_t _src_attr;
+  attr_t _weights_attr;
+  attr_t _bias_attr;
+
+  inner_product_forward_params() {}
+
+  inner_product_forward_params(
+      dnnl::inner_product_forward::primitive_desc&& pd,
+      dnnl::inner_product_forward&& primitive,
+      attr_t&& op_attr,
+      attr_t&& src_attr,
+      attr_t&& weights_attr,
+      attr_t&& bias_attr)
+      : _pd(pd),
```

Comment:
```suggestion
      : _pd(std::move(pd)),
```
```suggestion
      : _pd(pd),
```

Reply:
Suggest same changes to similar code below.

Reply:
Thanks. It's fixed. Please review again.